pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/RebalanceConfig.java

```diff
@@ -83,6 +83,13 @@ public class RebalanceConfig {
   @ApiModelProperty(example = "false")
   private boolean _bestEfforts = false;
 
+  // Whether to enforce Minimal Data Movement Algorithm (only effective if instance assignment config is set, and if
+  // bootstrap is false). If set to false, the minimizeDataMovement flag in the table config will be used to determine
```

Comment:
nit: update comment, that if set to DEFAULT we fallback on table config, otherwise use this as the override value